backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -493,6 +493,19 @@ static void maskToBitCount(Uint32 mask, uint8 &numBits, uint8 &shift) {
 }
 #endif
 
+void SurfaceSdlGraphicsManager::convertSDLPixelFormat(SDL_PixelFormat *in, Graphics::PixelFormat *out) const {
+	*out = Graphics::PixelFormat(in->BytesPerPixel,
+		8 - in->Rloss, 8 - in->Gloss,
+		8 - in->Bloss, 8 - in->Aloss,
+		in->Rshift, in->Gshift,
+		in->Bshift, in->Ashift);
+
+	// Workaround to SDL not providing an accurate Aloss value on Mac OS X.
+	if (in->Amask == 0)
```

Comment:
IMO this hack should remain in the one case where it is needed so it can be reassessed and eventually removed when reworking that code.